pkg/cmd/stack/translate.go

```diff
@@ -108,7 +108,8 @@ func translateConfigMap(s *model.Stack) *apiv1.ConfigMap {
 		ObjectMeta: metav1.ObjectMeta{
 			Name: model.GetStackConfigMapName(s.Name),
 			Labels: map[string]string{
-				model.StackLabel: "true",
+				model.StackLabel:      "true",
+				model.DeployedByLabel: format.ResourceK8sMetaString(s.Name),
```

Comment:
As the compose is not deployed through the proxy anymore, we add the deployed-by label in the compose translation